src/simulations/inversions/Inversion.jl

```diff
@@ -1,4 +1,63 @@
+export Inversion
+
+mutable struct InversionMetrics{F <: Real}
```

Comment:
I created this new object with all of the "repeated" information, next to the "new" information to make sure it can easily be used for analyzing through the plotting utilities. So you can easily plot any of the matrices that you want, together with rgi_id and longitude & latitude.

Reply:
I already created an object of type `Result` for this purpose. Most of these fields are already available either in the `parameters` or in the `glaciers`. I would really avoid repeating all this information. You can access everything within a `simulation`, so it is self contained enough. Can you please implement this with a `Result`? You can add a few fields there if they are not present anywhere else inside a `simulation`.

Reply:
But this `Results` isn't it meant for just running a forward model? I think it will be really confusing to add fields like:
`H_diff`, `V_diff`, `H_obs`, `V_obs`, mixing with the fields that are already there. Also currently in the whole frame work it is assumed that `A`,`n` and `C`. Are single numbers rather than matrices, which will make it non trivial to add it to `Results` or `Simulation` . In my opinion it would make more sense to make a new `Results`, which I called `inversionMetrics` in my case that contains everything that is important to inversions. I agree that there is some repetiveness here, however it makes it more clear in my opinion.

Currently the plotting utilities are designed in such  a way that it needs to be utilized as follow, where `ss` is the inversion object that contains` InversionMetrics` for each glacier. 

`plot_glacier(ss[glacier_id], "heatmaps", [:C,:H_pred,:V_obs])`

I think in this case it is just not trivial to just recycle the `Results` object and just add some fields.

Reply:
Please let me know if you have some insight that I am not seeing here

Reply:
This is what is currently present in a `Result`:
```julia

mutable struct Results{F <: AbstractFloat} 
    rgi_id::String
    H::Vector{Matrix{F}}
    S::Matrix{F}
    B::Matrix{F}
    V::Matrix{F}
    Vx::Matrix{F}
    Vy::Matrix{F}
    Δx::F             
    Δy::F  
    lon::F 
    lat::F          
end
```

OK, I think that the best way to go is the following one. You can make a new abstract type called `AbstractResults`, and then convert the current `Result` into a `SimulationResults`, as a child of `AbstractResults`. And then, you convert your `InversionMetrics` into `InversionResults`, as well a child from `AbstractResults`. This makes things tidier and it allows you to keep what you created. Sounds good?

Reply:
This sounds like a good solution indeed. Thanks for the additional insight!

Reply:
Okay so for some reason when implementing this `SimulationResults` and  `AbstractResults` are undefined within the `ODINN` test environment. However note that they are accesible when just `using ODINN`. I can't find a solution for this problem. So for now I just turned `InversionMetrics` into `InversionResults`.